docs/source/top_level_elements/language.rst

```diff
@@ -0,0 +1,172 @@
+
+https://digital.lib.utk.edu/collections/islandora/object/tatum%3A188/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm authority="iso639-2b" type="text">English</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:language <http://id.loc.gov/vocabulary/iso639-2/eng> .
+
+language has languageTerm subelement with an @authority of "iso639-2b" and a @type of "code"
+--------------------------------------------------------------------------------------------
+
+https://digital.lib.utk.edu/collections/islandora/object/ekcd:9/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm authority="iso639-2b" type="code">eng</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:language <http://id.loc.gov/vocabulary/iso639-2/eng> .
+
+language has multiple elements
+------------------------------
+
+Present for 11 total in utsmc
+
+https://digital.lib.utk.edu/collections/islandora/object/utsmc:725/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm authority="iso639-2b" type="text">French</languageTerm>
+    </language>
+    <language>
+        <languageTerm authority="iso639-2b" type="text">Italian</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1>
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/fre> ;
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/ita> .
+
+Present for 1 total in egypt
+
+https://digital.lib.utk.edu/collections/islandora/object/egypt:59/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm type="code" authority="iso639-2b">eng</languageTerm>
+    </language>
+    <language>
+        <languageTerm type="code" authority="iso639-2b">fre</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1>
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/eng> ;
+        dcterms:language <http://id.loc.gov/vocabulary/iso639-2/fre> .
+
+language has language subelement with an @type of #text and value of "No linguistic content"
+--------------------------------------------------------------------------------------------
+
+https://digital.lib.utk.edu/collections/islandora/object/tdh:911/datastream/MODS/view
+
+.. code-block:: xml
+
+    <language>
+        <languageTerm authority="iso639-2b" type="text">No linguistic content</languageTerm>
+    </language>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:language <http://id.loc.gov/vocabulary/iso639-2/zxx> ;
```

Comment:
You need a closing period instead of a semicolon.

Reply:
Addressed syntax issue in 6d1fdfe